pandas/tests/test_series.py

```diff
@@ -3633,12 +3636,12 @@ def test_timedelta64_conversions(self):
 
                 if m == 1 and unit != 'ns':
 
-                    # astype
+                    # a1
                     result = s1.astype("timedelta64[{0}]".format(unit))
                     assert_series_equal(result, expected)
 
                 # reverse op
-                expected = s1.apply(lambda x: np.timedelta64(m,unit) / x)
+                expected = s1.apply(lambda x: Timedelta(np.timedelta64(m,unit)) / x)
```

Comment:
should you just add a test here? (e.g. for both wrapped ``np.timedelta64`` and ``Timedelta``)